src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -18,36 +19,31 @@ internal void InitializeLicense(string licenseText, string licenseFilePath)
             var licenseSources = LicenseSources.GetLicenseSources(licenseText, licenseFilePath);
 
             var result = ActiveLicense.Find("NServiceBus", licenseSources);
+
+            var report = new StringBuilder();
+            report.AppendLine("Looking for license in the following locations:");
+
+            foreach (var item in result.Report)
+            {
+                report.AppendLine(item);
+            }
+
+            Logger.Info(report.ToString());
```

Comment:
I think this is to much noice since INFO is our default log level, how this:

if license found => log where we found it?

if not found => log all the places we considered?